theme/src/css/_zendesk-variables.scss

```diff
@@ -40,15 +40,16 @@ $font-family-base: "Source Sans Pro", "Source Serif Pro", -apple-system,
 $font-weight-light: 200 !default;
 $font-weight-semi-bold: 600 !default;
 
-$normal-font-size: 1rem !default; // 16px
-$small-font-size: 0.75rem !default; // 12px
+$font-size-default: 1rem !default; // 16px
+$font-size-md: 0.875rem !default; // 14px
+$font-size-sm: 0.75rem !default; // 12px
```

Comment:
@antoniorhee We can't rename `$small-font-size` here as it's overriding the bootstrap variable of the same name. Replacing it could introduce regressions. See https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/bootstrap/_variables.scss#L277

Reply:
Also notice on L34, there is a css comment indicating the variables in this section are "Variable overrides". For future Zendesk prs please send to me and request review with @jonmak08. See https://help.github.com/articles/requesting-a-pull-request-review/ Thanks!

Reply:
I told him to send it to me to review since you were out and I didn't want to hold up anything.  We can revert since this hasn't been released to Zendesk or anything.

Please advise @rdai10.  Thanks!

Reply:
Nah not a big deal, we can keep the current change upstream and top off the new changes when it comes in.

Reply:
Sounds good.  Thanks @rdai10!  We haven't released any updates to Zendesk so we can make whatever changes we need before the next release.

Reply:
Thanks for providing input @rdai10! Will be sending you a new PR soon.